.azure-pipelines/client.test.live.yml

```diff
@@ -0,0 +1,47 @@
+jobs:
+  - job: 'Test'
+    strategy:
+      matrix:
+        Linux_Python37:
+          OSVmImage: 'ubuntu-16.04'
+          PythonVersion: '3.7'
+        #Linux_Python27:
+        #OSVmImage: 'ubuntu-16.04'
+        #PythonVersion: '2.7'
+        #Windows_Python37:
+        #OSVmImage: 'vs2017-win2016'
+        #PythonVersion: '3.7'
+        #macOS_Python37:
+        #OSVmImage: 'macOS-10.13'
+        #PythonVersion: '3.7'
+      maxParallel: 1
+    continueOnError: false
+
+    pool:
+      vmImage: $(OSVmImage)
+
+    steps:
+      - task: UsePythonVersion@0
+        displayName: 'Use Python $(PythonVersion)'
+        inputs:
+          versionSpec: $(PythonVersion)
+
+      - script: |
+          python -m pip install --upgrade pip
+          python scripts/dev_setup.py --packageList azure-servicebus
```

Comment:
Move this to a build variable like the other one. In the build definition set the variable to `azure-servicebus` by default. As we enable more service's livetests, we can add them there.